app/boards/shields/zodiark/Kconfig.defconfig

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2021 Spencer Deven
```

Comment:
We always use the generic "The ZMK Contributors" for our copyrights, to avoid churn when files get changed by multiple users, etc., if you don't mind updating everywhere.

Reply:
Changed all files to reflect the correct copyright convention.